pages/_app.js

```diff
@@ -9,6 +10,7 @@ import breakpoints from 'common/styles/breakpoints';
 import Nav from 'components/Nav/Nav';
 import Footer from 'components/Footer/Footer';
 import 'common/styles/globalStyles.css';
+import withFonts from '../containers/withFonts/withFonts';
```

Comment:
I like the idea of not having the font loading decorator be outside of the components folder, but maybe we put it into a decorators folder instead of containers? What do you think?

Also, I was about to ask you to add the folder to a module resolver, but I think I'll change them in a later PR so that we dont have to do a new one for every root-level folder. We'll just resolve root at `~` 👍

Reply:
> I like the idea of not having the font loading decorator be outside of the components folder, but maybe we put it into a decorators folder instead of containers? What do you think?

Yep, sounds good to me, I can switch it up.

> Also, I was about to ask you to add the folder to a module resolver, but I think I'll change them in a later PR so that we dont have to do a new one for every root-level folder. We'll just resolve root at ~ 👍

👍